src/Arcus.Messaging.Tests.Integration/appsettings.local.json

```diff
@@ -2,6 +2,16 @@
   "Arcus": {
     "Health": {
       "Port": 42063
-    } 
-  } 
+    },
```

Comment:
Hmm, this was supposed to be ignore by the `.gitignore`, no?

Reply:
I did my best in the gitignore but I'm a rookie on that front.